### PR TITLE
fix(scylla_coredump_setup): remove the coredump create by the check

### PR DESCRIPTION
**Pull request by fruch**
since those coredumps are kept after the setup, this make out testing infra
to identify it as a coredump create by the test and fail it.

once it's working we can delete this coredump

Fixes: #6159

### CODE REVIEW

**Comment by fruch** [comment]
@penberg, this fails now the AMI test, and both master and enterprise are blocked by this

**Comment by bentsi** [comment]
this should fix #6159

File: dist/common/scripts/scylla_coredump_setup [hunk 1]
```diff
@@ -99,6 +99,7 @@ WantedBy=multi-user.target
         try:
             run('coredumpctl --no-pager --no-legend info {}'.format(pid))
             print('\nsystemd-coredump is working finely.')
+            run('rm /var/lib/systemd/coredump/core.bash.*')
```
Comment (bentsi) [inline dist/common/scripts/scylla_coredump_setup:102]: better `rm -f `

**Comment by roydahan** [comment]
@eliransin FYI

File: dist/common/scripts/scylla_coredump_setup (hunk 1 above)
Comment (eliransin) [inline dist/common/scripts/scylla_coredump_setup:102]: I think it is better to locate the file to delete the way cluster.py at sdcm does it.
see _notify_backtrace function.

Comment (amoskong) [reply dist/common/scripts/scylla_coredump_setup:102]: > I think it is better to locate the file to delete the way cluster.py at sdcm does it.
> see _notify_backtrace function.

It's not just a problem in testing env, we should not leave a meaningless core file to users, it's not graceful.

Comment (amoskong) [reply dist/common/scripts/scylla_coredump_setup:102]: @fruch do you agree to use '-f' option?

Comment (fruch) [reply dist/common/scripts/scylla_coredump_setup]: I agree

File: dist/common/scripts/scylla_coredump_setup [hunk 2]
```diff
@@ -99,6 +99,7 @@ WantedBy=multi-user.target
         try:
             run('coredumpctl --no-pager --no-legend info {}'.format(pid))
             print('\nsystemd-coredump is working finely.')
+            run('rm -f /var/lib/systemd/coredump/core.bash.*')
```
Comment (eliransin) [inline dist/common/scripts/scylla_coredump_setup:102]: Who said that all core.bash* are meaningless, I think I was misunderstood, I think we should delete **only**
the cores that were created in this script or we might delete unrelated legitimate cores.

**Comment by slivne** [comment]
@fruch ping - can you please address @eliransin comment above we need this merged for 4.1

**Comment by fruch** [comment]
I've address @eliransin concerns in 62a213e,
I've opened PR so we can stop doing this step in SCT, if we want something more of of it, that won't need to be in a different PR/patchset

**Comment by eliransin** [comment]
LGTM thanks @fruch

**Comment by eliransin** [comment]
@penberg this one is just pending merge (and your approval of course :slightly_smiling_face:

**Comment by penberg** [comment]
Queued to `next` as commit 28c3d4f8e8f7f13da1c28bcaa135d312e87d2d80.

**Pull request closed without merging**